pkg/reconciler/kubernetes/updatestrategyresolver_test.go

```diff
@@ -25,6 +26,8 @@ type testCase struct {
 }
 
 func TestDefaultUpdateStrategyResolver_Resolve(t *testing.T) {
+	t.Skip("mocking kube client test, can be removed.")
```

Comment:
This complete test can be removed because:
- Current {updatestrategyresolver.go} is always return {PatchUpdateStrategy}, it is meaningless to design a test at the moment.
- In order to construct the resolver, we have to mock the {resource.Helper} behavior, but with the new implementation, the helmClient can only accept {resource.Info}, and the helper can create by resource.Info member: {Client}, {Mapping} concretely, don't need to mock it.
{func NewHelper(client RESTClient, mapping *meta.RESTMapping) *Helper}